src/apps/filters/h3.c

```diff
@@ -35,47 +35,51 @@
 #include "h3Index.h"
 #include "utility.h"
 
+#define PARSE()                                                             \
+    if (parseArgs(argc, argv, sizeof(args) / sizeof(Arg *), args, &helpArg, \
+                  args[0]->helpText)) {                                     \
```

Comment:
I'd prefer to pass in argc, argv, etc. as macro arguments to make it clear those variables are getting used when the macro is called.

I'd also suggest adding a comment here that explicitly goes over some preconditions of this macro (there should be at least one arg, the subcommand arg should be first, etc)